library/Mockery/Adapter/Phpunit/MockeryTestCase.php

```diff
@@ -20,7 +20,44 @@
 
 namespace Mockery\Adapter\Phpunit;
 
+if (\PHP_VERSION_ID < 70000) {
```

Comment:
This might break existing PHPUnit 6 and 7 tests, PHPUnit 8+ should be explicitly targeted instead.